specs/Edge.md

```diff
@@ -0,0 +1,26 @@
+# Edge
+
+## Introduction
+The [Edge](https://github.com/openvstorage/home/wiki/Edge) is a lightweight software component which exposes a block device.
+
+
+## Design choices
+- No Edge manager is required
+ - This means we can't list all disks connected to certain host without retrieving the whole list
+- If possible don't model the edge
+
+
+## Required changes
+- Implement storage and management IP correctly
+    - DTL and MDS should use the management Ip
```

Comment:
Best practice is to configure storage ip on the network interface with the lowest latency, which is what you want for DTL & MDS.